### DISCUSSION TITLE
programme concour blanc

### DISCUSSION THREAD

**Discussion by Ke0li** [mp2i-fsm/mp2i-2021#125] [Question] [answered]
Bonjour Monsieur,

Est ce que tout les algorithmes sur les piles / files /tableau dynamique / dictionnaire   sont a connaitre?

Cordialement

Eloi

**Comment by fortierq** [comment, marked as answer]
Bonjour,

Il faut connaître les opérations qu'on a le droit de faire dessus et les implémentations possibles.

**Reply by fortierq** [reply]
Et s'entraîner avec les exercices.

**Reply by Ke0li** [reply]
Ça marche merci de votre réponse
